packages/genesys-spark-components/src/components/stable/gux-pagination/gux-pagination-buttons/gux-pagination-buttons.scss

```diff
@@ -65,6 +65,11 @@ gux-pagination-buttons {
             var(--gse-semantic-color-focus);
           outline-offset: var(--gse-semantic-focusRing-offset);
         }
+
+        &:disabled {
+          pointer-events: none;
+          opacity: 0.5; // TODO: Use token
```

Comment:
This version will be in the WCL (potentially) but not in Figma. In that case, is it necessary to use a token here or how does that work? @daragh-king-genesys @conor-darcy  any thoughts?